src/community/resources-and-tools/index.md.njk

```diff
@@ -17,6 +17,9 @@ The GOV.UK Design System team is not responsible for these resources and tools a
 [GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams) -
 A Sketch file for creating flow diagrams of GOV.UK services.
 
+[GOV.UK Design System Flow Diagrams for Miro](https://github.com/paulmsmith/govuk-designsystem-flow-diagram-miro) -
```

Comment:
I wasn't quite sure how to name my version. I based it on Dave House's version but the others seem to be called 'GOV Flow'.